examples/flux/readme.md

```diff
@@ -0,0 +1,32 @@
+# Deploy Flux1 Schnell on AWS EC2
+
+See a more [rich explanation](https://www.run.house/examples/host-and-run-flux1-image-genai-aws)
```

Comment:
If this isn't going in Examples, then this needs to change ☝️

Reply:
Good call! Changed to guides